drivers/console/multi_console.c

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright (c) 2018, ARM Limited and Contributors. All rights reserved.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#include <assert.h>
+#include <drivers/console.h>
+
+console_t *console_list;
+uint8_t console_state = CONSOLE_FLAG_BOOT;
+
+int console_register(console_t *console);	/* for use from asm only */
```

Comment:
Why do we need a declaration here?

Reply:
Oh, right, guess we don't. I'm used to writing code for projects that set -Wmissing-prototypes. Will fix.